tests/experimental/starry/light_curve_test.py

```diff
@@ -119,24 +125,24 @@ def test_compare_starry_system(keplerian_system):
             udeg=central.map.udeg,
             inc=np.rad2deg(central.map.inc),
             obl=np.rad2deg(central.map.obl),
-            amp=central.map.y[(0, 0)],
+            amp=central.map.amplitude,
         ),
         r=central.radius.magnitude,
         m=central.mass.magnitude,
         prot=central.map.period,
     )
     if central.map.u:
         pri.map[1:] = central.map.u
-
-    pri.map[1:, :] = central.map.y.todense().__array__()[1:]
+    if central.map.deg > 0:
+        pri.map[1:, :] = central.map.y.todense().__array__()[1:]
```

Comment:
Is this call to `__array__` to convert to numpy arrays? I normally use something like `np.asarray` because I think it's clearer...